web/client/selectors/context.js

```diff
@@ -64,22 +64,26 @@ export const templatesSelector = createSelector(
  *
  * @param {object} state the application state
  */
-export const pluginsSelector = state =>
-    isLoadingSelector(state)
-        ? loadingPluginsSelector(state)
-        : currentPluginsSelector(state) || defaultPluginsSelector(state);
+export const pluginsSelector = state => {
+    const isloading = isLoadingSelector(state);
+    const loadingPlugins = loadingPluginsSelector(state);
+    const defaultPlugins = defaultPluginsSelector(state);
+    const currentPlugins = currentPluginsSelector(state);
+    return isloading ? loadingPlugins : currentPlugins || defaultPlugins;
+};
+
 
 /**
  * Creates a selector that will return true if mapstore currently has a context active,
  * and it has a specific plugin activated
  * @param {string} pluginName plugin name
  */
-export const isPluginInContext = pluginName => createSelector(
-    currentContextSelector,
-    pluginsSelector,
-    (currentContext, contextPlugins = {}) => !currentContext ||
-        findIndex(get(contextPlugins, 'desktop', []), plugin => plugin.name === pluginName) > -1
-);
+export const isPluginInContext = pluginName => state => {
+    const currentContext = currentContextSelector(state);
+    const contextPlugins = pluginsSelector(state);
+    const result = !currentContext || findIndex(get(contextPlugins, 'desktop', []), plugin => plugin.name === pluginName) > -1;
+    return result;
```

Comment:
```suggestion
    return !currentContext || findIndex(get(contextPlugins, 'desktop', []), plugin => plugin.name === pluginName) > -1;
```

Reply:
for debugging purposes i used the committed solution we can keep it i guess